src/logic/Calculate.js

```diff
@@ -0,0 +1,112 @@
+import operate from './operate';
+
+function isNumber(item) {
+  return !!item.match(/[0-9]+/);
+}
+
+
+export default function calculate(obj, buttonName) {
+  if (buttonName === 'AC') {
```

Comment:
* Kindly fix the `AC` button as it doesn't clear the input field unless i refresh the browser 🙏